framework/decode/vulkan_object_info_table_base.h

```diff
@@ -32,11 +32,78 @@
 
 #include <cassert>
 #include <functional>
+#include <type_traits>
 #include <unordered_map>
 
 GFXRECON_BEGIN_NAMESPACE(gfxrecon)
 GFXRECON_BEGIN_NAMESPACE(decode)
 
+// NOTE: There's nothing VulkanSpecific in these utilities
```

Comment:
@MarkY-LunarG  -- this is the refactor to deal with the "specialization" limit that shows up under GCC.